component-samples/demo/scripts/keys_gen.sh

```diff
@@ -51,8 +51,8 @@ gen_credentials()
 api_user='C=US, ST=OR, L=Hillsboro, O=LF Edge, OU=FDO project, CN=apiUser'
 db_user=root
 api_pass=default
-db_password=`openssl rand --base64 12 | tr -dc 0-9A-Za-z`
-encrypt_password=`openssl rand --base64 12 | tr -dc 0-9A-Za-z`
+db_password=`openssl rand -base64 12 | tr -dc 0-9A-Za-z`
+encrypt_password=`openssl rand -base64 12 | tr -dc 0-9A-Za-z`
```

Comment:
--base also works right ?

Reply:
Doesn't  work anymore, it gives info about proper usage of 'rand' command.